.local/uuid/abc.js

```diff
@@ -0,0 +1 @@
+sdfkl;
```

Comment:
Should this be committed?

Reply:
Oops, of course not…. I was Troubleshooting eslint/prettier… will remove